src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx

```diff
@@ -128,6 +128,11 @@ const StyledNote = styled(TitleM)`
   overflow: hidden;
   line-clamp: 4;
   display: -webkit-box;
+  text-overflow: ellipsis;
+
+  /* For safari: we need to specify the height */
+  /* Default font size * line-height * line-clamp */
+  height: calc(1rem * 1.5 * 4);
```

Comment:
@jakzaizzat can you link to a stack overflow or something? why do we need to specify this height?

Reply:
@tbezman Seems like the `overflow hidden` didnt work very well in the new safari

https://stackoverflow.com/questions/70679732/css-line-clamp-does-not-work-in-safari-on-inner-block-level-elements/70683084#70683084

Found gist on line clamp safari

https://gist.github.com/EminQasimov/80ddcb7289d47a4d07caa1b4dc2bae83

Reply:
let's leave this in the code

Reply:
Updated! Found other solution without using this hacky height stuff